doc/jemalloc.xml.in

```diff
@@ -1544,8 +1573,24 @@ malloc_conf = "xmalloc:true";]]></programlisting>
         set, all currently unused dirty pages are considered to have fully
         decayed, which causes immediate purging of all unused dirty pages unless
         the decay time is set to -1 (i.e. purging disabled).  See <link
-        linkend="opt.decay_time"><mallctl>opt.decay_time</mallctl></link> for
-        additional information.</para></listitem>
+        linkend="opt.dirty_decay_time"><mallctl>opt.dirty_decay_time</mallctl></link>
+        for additional information.</para></listitem>
+      </varlistentry>
+
+      <varlistentry id="arena.i.muzzy_decay_time">
```

Comment:
I found "muzzy" as (IMO) the least bad name after an arduous thesaurus session...